SparkPost/2web2ui
# FE-1401: Removed unused state variables

### What Changed

- Removed unused state variables in api keys page and tfa page

### How To Test

-Go to API key page, create api key click copy
- Verify that copied messaging still appears
- On new account create a new user
- Enable required TFA in account settings
- Log into new user
- Verify that TFA initial for still works as intended

### To Do
